tools/test/test_depsolve.py

```diff
@@ -40,9 +41,7 @@ def depsolve(pkgs, repos, root_dir, cache_dir, command):
             ]
         }
     }
-    p = sp.run([command], input=json.dumps(req).encode(), check=True, capture_output=True)
-    if p.stderr:
-        print(p.stderr.decode())
+    p = sp.run([command], input=json.dumps(req).encode(), check=True, stdout=sp.PIPE, stderr=sys.stderr)
```

Comment:
So it will suppress stderror if it will be presented?

Reply:
Yes, sorry, I was confused by my own patch :( So the issue here is that the `print(p.stderr.decode())` will never run if the exit code is != 0 because with `check=True` that will cause an exception (and the output of stderr is not part of the exception). So the easiest fix here seems to just always directly print stderr to the our own (sys.stderr)

Reply:
Thanks for the explanation:)